### ISSUE TITLE
Race between `_FullScreen` and other screen-related commands

### ISSUE THREAD

**Issue by mkilgore** [QB64-Phoenix-Edition/QB64pe#292]
After the completion of #66 most commands that involve GLUT do not wait for the command to complete before returning, rather they simply add work to the GLUT command queue that will be done later on the GLUT thread at the next interval (typically 1/60 of a second). We don't bother waiting because many commands do not return anything or have any program-noticeable side-effects, so it's just not necessary.

This creates a problem with `_FullScreen` because although it really is a GLUT-based command it was not implemented by calling GLUT directly but rather by setting some global variables that the GLUT thread checks. This is a bit of an ugly way to do it, but also means that it happens independent of the GLUT command queue mentioned earlier, and some commands like `_SCREENMOVE` cause the program to exit fullscreen. Thus, sequences that should work like having `_SCREENMOVE _MIDDLE` before `_FULLSCREEN` actually don't, because the fullscreen can get handled before the screenmove (which then results in the screenmove exiting the fullscreen).

The solution here should be pretty simple, `sub__fullscreen` should not be setting global variables but instead should add a new 'fullscreen' command to the GLUT queue. That command will be executed in-order with all the other GLUT commands in the queue and things will work as intended.

**Comment by Raven-Singularity** [comment]
I see the switch from FreeGLUT to GLFW is underway by the linked page.

Is it correct that the switch to GLFW has not yet made it into the public versions of QB64-PE?

Seems like a massive internal change, there's about 100 items on that page!

**Comment by a740g** [comment]
> I see the switch from FreeGLUT to GLFW is underway by the linked page.
> 
> Is it correct that the switch to GLFW has not yet made it into the public versions of QB64-PE?
> 
> Seems like a massive internal change, there's about 100 items on that page!

Yes, it isn't public yet. We're very close to merging the GLFW PR into the main branch and may publish a preview release shortly. Please test it and share any feedback.
